src/Application.cc

```diff
@@ -86,6 +86,35 @@ Application::Application(int &_argc, char **_argv, const WindowType _type)
 {
   igndbg << "Initializing application." << std::endl;
 
```

Comment:
We need to specify the choice of render system at this point so we can choose the appropriate Qt scene graph backend (to use the Ogre2 Metal render system we must use the matching `QSGRendererInterface::MetalRhi` backend).

For the MinimalScene plugin the render system can be set in the plugin XML, however that is not available to use at this point, so we need a command line option or some other configuration mechanism.

Reply:
@iche033 suggested here https://github.com/ignitionrobotics/ign-rendering/pull/498#issuecomment-984161539 that we default to the Metal graphics API when the platform is `__APPLE__`. This would address this question and similar ones for ign-gazebo. Given the limited support for OpenGL on macOS I can't see the gui tools working on macSO with ogre2 / OpenGL in the immediate future (I've decided to abandon my efforts to try and retrofit OpenGL 4.1 support in ogre-nextv2.2).